relay-log/src/setup.rs

```diff
@@ -317,6 +329,14 @@ pub fn init(config: &LogConfig, sentry: &SentryConfig) {
             }));
         }
 
+        crate::info!(
+            release = RELEASE,
+            server_name = ?sentry.server_name,
+            environment = ?sentry.environment,
```

Comment:
I would love these fields to be added conditionally (only if they're `Some`), but `tracing` doesn't seem to support that.

Reply:
That should be the default behaviour: https://docs.rs/tracing-core/0.1.32/src/tracing_core/field.rs.html#749

Reply:
Ah maybe doesn't work here because you convert to `Debug` representation with the `?`, if you drop it, it should work.

Reply:
I misinterpreted the error message. I though `Option` didn't implement `Value`, but actually `Cow<str>` is the problem.